Bruin Bite/Models/ErrorLogger.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ErrorLogger.swift
+//  Bruin Bite
+//
+//  Created by Kameron Carr on 9/1/18.
+//  Copyright © 2018 Dont Eat Alone. All rights reserved.
+//
+
+import Foundation
+
+enum ErrorType: String {
+    case error = "❤️ -ERROR- ❤️"
```

Comment:
Add documentation comments above each of these cases, such that XCode's autocomplete can help callers of the function will know when to use what log level. (the comments being added in will be displayed along with all the different available enum options in the autocomplete menu).
```swift
/** Message describing an error that causes catastrophic failure. **/
case error = "❤️ -ERROR- ❤️"
/** Message describing an error that causes a minor inconvenience. **/
case warning = "💛 -WARNING- 💛"
/** Message describing some general information. **/
case info = "💚 -INFO- 💚"
/** Messages used solely for debug purposes. All DEBUG logs will be removed before pushing to production. **/
case debug = "💙 -DEBUG- 💙"
/** Lengthy messages used for detailed descriptions. **/
case verbose = "💜 -VERBOSE- 💜"
/** Messages that don't fall under any of these categories (be very sure of this) **/
case other = "🖤 -OTHER- 🖤"
/** Messages that don't want headers before their log outputs. **/
case none = ""
```